fix: fix alignment of colorbar labels for categorical color column in parallel coordinates & parallel categories sections

mbelak-dtml opened the pull request:
The first and tick labels for categorical columns would previously be aligned to the ends of the colorbar. This PR ensures that the edge tick labels are aligned to the center, same as the non-edge ones are aligned.

lukany reviewed (changes requested):
1. I would add more details to the commit message as the changelog is generated from commit messages and the current text could be ambiguous. I would mention that it affects parallel coordinates and parallel categories plots in multivariate analysis.
2.  Not sure if it's `feat` or `fix`. Personally it feels more like a `fix` but leaving it up to you.

mbelak-dtml commented:
> 1. I would add more details to the commit message as the changelog is generated from commit messages and the current text could be ambiguous. I would mention that it affects parallel coordinates and parallel categories plots in multivariate analysis.
> 2. Not sure if it's `feat` or `fix`. Personally it feels more like a `fix` but leaving it up to you.



> 1. I would add more details to the commit message as the changelog is generated from commit messages and the current text could be ambiguous. I would mention that it affects parallel coordinates and parallel categories plots in multivariate analysis.
> 2. Not sure if it's `feat` or `fix`. Personally it feels more like a `fix` but leaving it up to you.

Updated the PR title with mention to which plots it affects.

Changed to `fix`. I'm also not really sure though. Could be seen either way IMO. Fix as in fixing a plot that looks bad by default. Or adding a feature on top of plotly defaults to make the plot look better.
